### ISSUE TITLE
Incorrect conversion of double bar to absolute value. 

### ISSUE THREAD

**Issue by nirupa-kumar** [zorkow/speech-rule-engine#20]
When we submit the below equation for conversion : 
‖x+1‖  - the text returned is "StartAbsoluteValue x plus 1 EndAbsoluteValue" 
The input submitted is in mml form : 
```javascript
<math display='block'>
 <mrow>
  <mrow><mo>‖</mo> <mrow>
   <mi>x</mi><mo>+</mo><mn>1</mn></mrow><mo>‖</mo></mrow></mrow>
</math>
```

As per our understanding , a single bar such as |x+1| would denote absolute value.  A possible text description for the above is "double bar" instead of absolute value. 
Please let me know if you need any further information.

**Comment by zorkow** [comment]
Thanks for reporting this. I should have time to look into this tomorrow,

**Comment by zorkow** [comment]
Fixed in version 0.2.7 available via npm.
